test/sub-menu.html

```diff
@@ -625,7 +625,7 @@
           const overlay = menu.$.overlay;
           overlay.__openingHandler && overlay.__openingHandler();
         }
-        MockInteractions.tap(openTarget, {emulateTouch: true});
+        openTarget.dispatchEvent(new CustomEvent('mouseover', {bubbles: true, composed: true}));
```

Comment:
Nit: probably it makes sense to get rid out of `tap` over here as well: https://github.com/vaadin/vaadin-menu-bar/blob/6f1cb55fbcb96c3097d6fe5053269501187b5912/test/sub-menu.html#L685 Then it will be possible to remove import of the `mock-interactions` in this file.

Reply:
done